draft-ietf-quic-transport.md

```diff
@@ -2211,7 +2211,9 @@ endpoint.  An endpoint that sends packets near the end of the idle timeout
 period of a peer risks having those packets discarded if its peer enters the
 draining state before the packets arrive.  If a peer could timeout within an
 Probe Timeout (PTO, see Section 6.2.2 of {{QUIC-RECOVERY}}), it is advisable to
-test for liveness before sending any data that cannot be retried safely.
+test for liveness before sending any data that cannot be retried safely.  Note
+that it is likely that only application protocols or using applications will
+know what information can be retried.
 
```

Comment:
what does "using applications" mean?

Reply:
Applications using the application protocol, presumably.  For instance, an HTTP library might not really know whether retrying something is safe -- it's the caller of the library that ultimately has to decide.